### PR TITLE
feat-14-modals

### PR DESCRIPTION
**Pull request by SinchanaMS**
Added the Modal component.

Files to Check:
1. [modals.html](https://github.com/SinchanaMS/Clay/pull/18/files#diff-8d34562e841d4e24f70059f609d6f15d815eff540f79ad69b5279101e0fe8bf2)
2. [modals.css](https://github.com/SinchanaMS/Clay/pull/18/files#diff-efe0bc3d069f7a6bd21acabaad99bed64b8f4665bd2e47ff6931b6bfbee8d009)

The rest of the files can be ignored, the page links were updated to include Modals.

Preview:
1. CTA on the right end of the Modal:

image

2. CTA at the centre of the Modal:

image

### CODE REVIEW

File: docs/modals.html [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+                    Use the class <code class="class-highlight">modal-cta</code> in the CTA buttons and you are good to
+                    go.
+                </div>
+
+                <div class="component-showcase-col modals">
+                    <div class="modal-bg-content"></div>
+                    <div class="modal">
+                        <h1>Order Placed!</h1>
+                        <hr />
+                        <p class="p-lg">Congratulations! Your order has been confirmed.</p>
```
Comment (dev-ashishsethi) [inline docs/modals.html:127]: improve class naming it should be self explanatory (just a suggestion)

File: docs/modals.html [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+            </div>
+
+        </section>
+    </div>
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/highlight.min.js"
+        integrity="sha512-IaaKO80nPNs5j+VLxd42eK/7sYuXQmr+fyywCNA0e+C6gtQnuCXNtORe9xR4LqGPz5U9VpH+ff41wKs/ZmC3iA=="
+        crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+    <script>
+        hljs.highlightAll();
```
Comment (dev-ashishsethi) [inline docs/modals.html:168]: good that you are using highlight.js

**Review by dev-ashishsethi — APPROVED** [review]
do check the comments

**Comment by shruthipatil11** [comment]
Code looks good to me

**Review by shruthipatil11 — APPROVED** [review]

**Comment by SinchanaMS** [comment]
Thank you for the review. Merging to temp-dev.

**Pull request merged**